src/main/java/seedu/address/model/fields/ApplicationOutcome.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.fields;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * An outcome for a job application
+ */
+public class ApplicationOutcome {
+
+    private static final List<String> possibleOutcomes =
+            new LinkedList<>(Arrays.asList("follow up", "ghosted", "offered", "rejected", "accepted"));
```

Comment:
is there a reason for this to be a LinkedList?

Reply:
Hmm, what do u suggest?

Reply:
nah, just wondering if there was a function of linkedlists you need to use